src/ontology/mondo-edit.obo

```diff
@@ -619468,6 +619466,29 @@ relationship: excluded_from_qc_check http://purl.obolibrary.org/obo/mondo/sparql
 relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/4585 {source="PMID:27683935"} ! GRIN2A
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9063" xsd:anyURI
 
+[Term]
+id: MONDO:1060144
+name: retinitis pigmentosa 7, digenic
+def: "A digenic form of retinitis pigmentosa resulting from a mutation in the PRPH2 gene and a null mutation of the ROM1 gene, leading to progressive degeneration of the retina and vision loss." [OMIM:608133]
+xref: OMIM:608133 {source="MONDO:includedEntryInOMIM"}
+is_a: MONDO:0019200 {source="OMIM:608133"} ! retinitis pigmentosa
+is_a: MONDO:1040055 {source="OMIM:608133"} ! PRPH2-related retinopathy
+relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/hgnc/10254 {source="OMIM:608133"} ! ROM1
```

Comment:
By definition (ie the disease is digenic), we need to have 2 annotations: one for the variation in PRPH2 gene, and one for the ROM1 gene. Please note the comment above about adding this term to not be updated as part of the automated update.